roles/olm_operator/tasks/main.yml

```diff
@@ -128,7 +127,7 @@
           apiVersion: operators.coreos.com/v1alpha1
           kind: Subscription
           metadata:
-            name: "{{ operator }}"
+            name: "{{ subscription_name | default(operator) }}"
```

Comment:
Thanks, please add this to the readme. Regards

Reply:
+1. Fixed